runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -367,6 +435,46 @@ private void hideWidget(Widget widget, boolean hidden)
 		}
 	}
 
+	private void sendChatMessage(String chatMessage)
+	{
+		final String message = new ChatMessageBuilder()
+			.append(ChatColorType.NORMAL)
+			.append(chatMessage)
+			.build();
+
+		chatMessageManager.queue(
+			QueuedMessage.builder()
+				.type(ChatMessageType.GAME)
+				.runeLiteFormattedMessage(message)
+				.build());
+	}
+
+	private String getAttackColor(Skill skill)
+	{
+		String color = "<col=ff0000>";
+		switch (skill)
+		{
+			case ATTACK:
+			case STRENGTH:
+			case DEFENCE:
+			{
+				color = "<col=D44A4A>";
+				break;
+			}
+			case RANGED:
+			{
+				color = "<col=259443>";
+				break;
+			}
+			case MAGIC:
+			{
+				color = "<col=369EB3>";
+				break;
+			}
+		}
+		return color + skill.getName() + "</col>";
```

Comment:
Now that #4072 has been merged, use `ColorUtil::wrapWithColorTag` and store an actual _`Color`_ object in `color` to change this to `return ColorUtil.wrapWithColorTag(skill.getName(), color);`